config/mongoid.yml

```diff
@@ -30,3 +30,5 @@ production:
         server_selection_timeout: 5
         write:
           w: <%= ENV['MONGO_WRITE_CONCERN'] || 'majority' %>
+        read:
+          mode: :primary_preferred
```

Comment:
should this be a symbol? I don't think they exist in YAML

Reply:
Hmm I was going off https://github.com/mongodb/mongoid/blob/master/lib/rails/generators/mongoid/config/templates/mongoid.yml#L163 I tested this in the rails console and that the rails app would start locally. I'll try with a string as this was how it was previously configured.

Reply:
It doesn't work as a string:
```
Loading development environment (Rails 5.2.0)
irb(main):001:0> ContentItem.count
KeyError: key not found: "primary_preferred"
	from (irb):1
```
whereas the config as symbol:
```
Loading development environment (Rails 5.2.0)
irb(main):001:0> ContentItem.count
=> 444830

Reply:
Well TIL - sorry Steve!